### ISSUE TITLE
Lack of focus trap in hamburger menu after X button

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#34]
[Aan de slag met API's](https://don.apps.digilab.network/kennisbank/apis/aan-de-slag/)

Na de "X" knop van het Hamburger menu gaat de focus naar items in de hoofdnavigatie erachter, buiten het hamburger menu. Pas nadat door alle items van de navigatie heen is gegaan, beweegt de focus terug naar binnen het hamburger menu.

`<button type="button" aria-label="Sluit navigatiebalk" class="clean-btn navbar-sidebar__close"><svg viewBox="0 0 15 15" width="21" height="21"><g stroke="var(--ifm-color-emphasis-600)" stroke-width="1.2"><path d="M.75.75l13.5 13.5M14.25.75L.75 14.25"></path></g></svg></button>`

WCAG 2.4.3 Focus order
WCAG 2.4.7 Focus visible

<img width="1138" alt="Expanded hamburger menu on a webpage where the focus is on an element behind the hamburger menu" src="https://github.com/user-attachments/assets/ed4a6059-232e-4297-8538-4edab0e5b9fd" />

_Reference: A020_
